payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json

```
@@ -0,0 +1,1728 @@
+										},
+										{
+											"key": "x-application-context",
+											"value": "identity-service:oauth-clients,one-time-token,staging,kubernetes-customenvs:12345"
+										},
+										{
+											"key": "cache-control",
+											"value": "no-store"
+										},
+										{
+											"key": "pragma",
+											"value": "no-cache"
+										},
+										{
+											"key": "x-content-type-options",
+											"value": "nosniff"
+										},
+										{
+											"key": "x-xss-protection",
+											"value": "1; mode=block"
+										},
+										{
+											"key": "x-frame-options",
+											"value": "DENY"
+										},
+										{
+											"key": "x-newrelic-app-data",
+											"value": "PxQGWVBaCwUGR1JTAQMPUFAHBRFORDQHUjZKA1ZLVVFHDFYPbU5kBwZqGA4HFkxbbEsVDVxUWUZMYXlgNk9BTgBMCE4IFgQKB1cLVQRZH1VRGhVXUAVcBghUU1oEBA9RAFUBEU4AAg5DB2U="
+										},
+										{
+											"key": "vary",
+											"value": "Accept-Encoding"
+										},
+										{
+											"key": "x-envoy-upstream-service-time",
+											"value": "91"
+										},
+										{
+											"key": "x-envoy-attempt-count",
+											"value": "1"
+										}
+									],
+									"cookie": [],
+									"body": "{\n    \"access_token\": \"{access_token}\",\n    \"token_type\": \"bearer\",\n    \"refresh_token\": \"{refresh_token}\",\n    \"expires_in\": 43199,\n    \"scope\": \"transfers\",\n    \"created_at\": \"2020-09-16T10:19:33.145563Z\"\n}"
+								}
+							]
+						}
+					],
+					"description": "After previosuly completing 1a or 1b you will have a refresh token stored, use this to generate new access tokens"
+				}
+			],
+			"event": [
+				{
+					"listen": "prerequest",
+					"script": {
+						"type": "text/javascript",
+						"exec": [
+							""
+						]
+					}
+				},
+				{
+					"listen": "test",
+					"script": {
+						"type": "text/javascript",
+						"exec": [
+							""
+						]
+					}
+				}
+			]
+		},
+		{
+			"name": "2. Transfer Flow",
+			"item": [
+				{
+					"name": "1. Create quote",
+					"item": [
+						{
+							"name": "2. Create Quote",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"var jsonData = JSON.parse(responseBody);",
+											"if (jsonData.id) postman.setEnvironmentVariable(\"new-quote-id\", jsonData.id);"
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "POST",
+								"header": [
+									{
+										"key": "Authorization",
+										"value": "Bearer {{token}}"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n\t\"sourceCurrency\": \"GBP\",\n\t\"targetCurrency\": \"USD\",\n\t\"sourceAmount\": 100,\n\t\"targetAmount\": null,\n\t\"profile\": <owning profile id>\n}"
```

```suggestion
									"raw": "{\n\t\"sourceCurrency\": \"GBP\",\n\t\"targetCurrency\": \"USD\",\n\t\"sourceAmount\": 100,\n\t\"targetAmount\": null,\n\t\"profile\": {{business-profile-id}}\n}"
```

This should now work, at the time there was a postman bug